packages/calcite-design-tokens/support/token-transformer/styleDictionary/formatter/utils.ts

```diff
@@ -1,14 +1,14 @@
 import { PlatformFormats, PlatformUnion } from "../../../types/platform.js";
 import { FormatterRules, MappedFormatterArguments } from "../../../types/styleDictionary/formatterArguments.js";
 import { CalciteCss } from "./css.js";
-import { CalciteJs } from "./javascript.js";
+import { CalciteJson } from "./json.js";
 import { CalciteScss } from "./scss.js";
 
 const formatters: Partial<Record<PlatformFormats, string>> = {
   css: CalciteCss,
   scss: CalciteScss,
   sass: CalciteScss,
-  js: CalciteJs,
+  json: CalciteJson,
```

Comment:
Do we need a custom formatter or can we use the predefined formatter for JSON?

Reply:
Did a shallow test and it looks like contents are identical.

Reply:
updated to use "json"